testsupport/init.go

```diff
@@ -67,6 +69,36 @@ func waitForOperators(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	cl.Create(context.TODO(), &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "e2e_test",
+			Namespace: hostNs}})
+
+	crb := rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "e2e_test:admin",
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "admin",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "e2e_test",
+				Namespace: hostNs,
+			},
+		},
+	}
+
+	if err := cl.Create(context.TODO(), &crb); err != nil {
+		fmt.Printf("error in creating rbac for service account %s", err)
+	}
+
+	bt := &auth.TokenRequest{}
+	kubeconfig.BearerToken = bt.Status.Token
```

Comment:
you need to send the TokenRequest first to get the token. In this way, the `bt.Status.Token` will be empty. You can use the function from https://github.com/codeready-toolchain/toolchain-common/blob/ad743a86ea5f9e9e01d1cfa24c64fa60de7f6394/pkg/client/rest_client.go

Reply:
yeah right.. updated it